apps/dialer/index.html

```diff
@@ -18,6 +18,9 @@
 
     <script src="js/config.js"></script>
     <script src="js/main.js"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Readex+Pro:wght@160..700&display=swap" rel="stylesheet">
```

Comment:
We can't use fonts hosted on remote servers because:
- that can be slow to load.
- that won't work at all if you want to use the dialer when you have no data or wifi connectivity.
- the files could change without you being aware.

If we really need a new font, it needs to be imported in the app itself. But you use `Readex Pro` which is already packaged in Nutria: https://github.com/capyloon/nutria/tree/main/apps/shared/fonts/Readex_Pro so you should point to that one instead.

Reply:
I tried to resolve that now by installing the font inside the style folder.